pkg/service/roomservice.go

```diff
@@ -207,10 +217,18 @@ func (s *RoomService) GetParticipant(ctx context.Context, req *livekit.RoomParti
 func (s *RoomService) RemoveParticipant(ctx context.Context, req *livekit.RoomParticipantIdentity) (*livekit.RemoveParticipantResponse, error) {
 	AppendLogFields(ctx, "room", req.Room, "participant", req.Identity)
 
+	if err := EnsureAdminPermission(ctx, livekit.RoomName(req.Room)); err != nil {
```

Comment:
this wasn't enforced before ?! 😱

Reply:
it was, it's checked later in the legacy rpc helper